h/views/client.py

```diff
@@ -85,6 +112,7 @@ def sidebar_app(request, extra=None):
     ga_client_tracking_id = settings.get('ga_client_tracking_id')
 
     ctx = _app_html_context(api_url=request.route_url('api.index'),
+                            client_boot_url=_resolve_client_boot_url(request),
```

Comment:
Is there a reason we need to actually resolve the URL here? Can we use the value of the setting without making any HTTP requests?